src/api/comments.ts

```diff
@@ -0,0 +1,10 @@
+import { Comment } from '../types/Comment';
+import { client } from '../utils/fetchClient';
+
+export const getComments = (postId: number) => {
+  return client.get<Comment[]>(`/comments?postId=${postId}`);
+};
+
+export const deleteComment = (commentId: number) => {
+  return client.delete(`/comments/${commentId}`);
```

Comment:
Consider adding error handling for the `deleteComment` API call. Adding a `.catch` block will help manage errors and provide feedback to the user if the deletion fails.